src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1232,14 +1232,14 @@ public function getMimeType($format)
     /**
      * Gets the format associated with the mime type.
      *
-     * @param string $mimeType The associated mime type
+     * @param string $extendedMimeType The associated mime type
      *
      * @return string|null The format (null if not found)
      */
-    public function getFormat($mimeType)
+    public function getFormat($extendedMimeType)
```

Comment:
I don't think it's necessary to change this variable name. `$mimeType` should be enough. It doesn't matter if the MIME type has parameters or not, both are MIME types.